<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>816</width>
    <height>724</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SaFri Player</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>img/sf_icon.png</normaloff>img/sf_icon.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-image:url(img/steel_background.jpg);</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <property name="margin">
     <number>2</number>
    </property>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,0">
      <property name="spacing">
       <number>0</number>
      </property>
      <item>
       <widget class="QWidget" name="leftWidget" native="true">
        <layout class="QVBoxLayout" name="verticalLayout" stretch="0,20">
         <item>
          <widget class="QTabWidget" name="treeViewTabWidget">
           <property name="contextMenuPolicy">
            <enum>Qt::CustomContextMenu</enum>
           </property>
           <property name="styleSheet">
            <string notr="true">background-image:url();</string>
           </property>
           <property name="currentIndex">
            <number>0</number>
           </property>
           <widget class="QWidget" name="webTab">
            <attribute name="title">
             <string>Seite</string>
            </attribute>
            <layout class="QHBoxLayout" name="horizontalLayout_6">
             <item>
              <widget class="QWebView" name="webView">
               <property name="url">
                <url>
                 <string>about:blank</string>
                </url>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </widget>
         </item>
         <item>
          <widget class="QTreeView" name="storedPlaylistView">
           <property name="styleSheet">
            <string notr="true">background-image:url();</string>
           </property>
           <property name="dragEnabled">
            <bool>false</bool>
           </property>
           <property name="dragDropMode">
            <enum>QAbstractItemView::NoDragDrop</enum>
           </property>
           <property name="rootIsDecorated">
            <bool>false</bool>
           </property>
           <attribute name="headerVisible">
            <bool>false</bool>
           </attribute>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="rightWidget" native="true">
        <layout class="QVBoxLayout" name="verticalLayout_3" stretch="0,82,0,3">
         <item>
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>19</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="PlaylistView" name="playlistView">
           <property name="styleSheet">
            <string notr="true">background-image:url();</string>
           </property>
           <property name="dragEnabled">
            <bool>false</bool>
           </property>
           <property name="dragDropOverwriteMode">
            <bool>true</bool>
           </property>
           <property name="dragDropMode">
            <enum>QAbstractItemView::DragDrop</enum>
           </property>
           <property name="defaultDropAction">
            <enum>Qt::MoveAction</enum>
           </property>
           <property name="alternatingRowColors">
            <bool>false</bool>
           </property>
           <property name="selectionMode">
            <enum>QAbstractItemView::SingleSelection</enum>
           </property>
           <property name="rootIsDecorated">
            <bool>false</bool>
           </property>
           <property name="itemsExpandable">
            <bool>false</bool>
           </property>
           <property name="sortingEnabled">
            <bool>false</bool>
           </property>
           <property name="animated">
            <bool>true</bool>
           </property>
           <property name="headerHidden">
            <bool>true</bool>
           </property>
           <attribute name="headerVisible">
            <bool>false</bool>
           </attribute>
           <attribute name="headerCascadingSectionResizes">
            <bool>false</bool>
           </attribute>
           <attribute name="headerShowSortIndicator" stdset="0">
            <bool>true</bool>
           </attribute>
          </widget>
         </item>
         <item>
          <widget class="QWidget" name="widget_2" native="true">
           <layout class="QVBoxLayout" name="verticalLayout_4">
            <property name="spacing">
             <number>0</number>
            </property>
            <property name="margin">
             <number>0</number>
            </property>
            <item>
             <widget class="PlayerWidget" name="playerWidget" native="true">
              <property name="autoFillBackground">
               <bool>true</bool>
              </property>
              <property name="styleSheet">
               <string notr="true">background-image:url();</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QWidget" name="widget" native="true">
           <property name="styleSheet">
            <string notr="true">background-image:url();</string>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_11">
            <property name="spacing">
             <number>3</number>
            </property>
            <property name="margin">
             <number>3</number>
            </property>
            <item>
             <widget class="QPushButton" name="actualPlayingPlaylistButton">
              <property name="text">
               <string>Aktuelle Playliste</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="clearPlaylistButton">
              <property name="text">
               <string>Playliste leeren</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="restorePlaylistButton">
              <property name="text">
               <string>Playliste wiederherstellen</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="savePlaylistButton">
              <property name="text">
               <string>Playliste speichern</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QProgressBar" name="progressBar">
      <property name="value">
       <number>24</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>816</width>
     <height>18</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image:url();</string>
   </property>
   <widget class="QMenu" name="menuSammlung">
    <property name="title">
     <string>Sammlung</string>
    </property>
    <addaction name="actionSongs_hinzufuegen"/>
    <addaction name="actionOrdner_hinzufuegen"/>
    <addaction name="actionBereinigen"/>
   </widget>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Einstellungen</string>
    </property>
    <addaction name="action_Open_Settings"/>
    <addaction name="action_about_SaFri_Player"/>
   </widget>
   <addaction name="menuSammlung"/>
   <addaction name="menu"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image:url();</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionSongs_hinzufuegen">
   <property name="icon">
    <iconset>
     <normaloff>img/addsong_icon.png</normaloff>img/addsong_icon.png</iconset>
   </property>
   <property name="text">
    <string>Songs hinzufügen</string>
   </property>
  </action>
  <action name="actionOrdner_hinzufuegen">
   <property name="icon">
    <iconset>
     <normaloff>img/addfolder_icon.png</normaloff>img/addfolder_icon.png</iconset>
   </property>
   <property name="text">
    <string>Ordner hinzufügen</string>
   </property>
  </action>
  <action name="actionSavePlaylist">
   <property name="text">
    <string>Playliste speichern</string>
   </property>
   <property name="toolTip">
    <string>Speichert die angezeigten Lieder in einer Playliste.</string>
   </property>
  </action>
  <action name="actionRestorePlaylist">
   <property name="text">
    <string>Wiederherstellen</string>
   </property>
   <property name="toolTip">
    <string>letzte Ansicht wiederherstellen</string>
   </property>
  </action>
  <action name="action_about_SaFri_Player">
   <property name="icon">
    <iconset>
     <normaloff>img/sf_icon.png</normaloff>img/sf_icon.png</iconset>
   </property>
   <property name="text">
    <string>Über SaFri Player</string>
   </property>
  </action>
  <action name="action_Open_Settings">
   <property name="text">
    <string>Einstellungen</string>
   </property>
  </action>
  <action name="actionBereinigen">
   <property name="text">
    <string>Bereinigen</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QWebView</class>
   <extends>QWidget</extends>
   <header>QtWebKit/QWebView</header>
  </customwidget>
  <customwidget>
   <class>PlayerWidget</class>
   <extends>QWidget</extends>
   <header location="global">playerwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>PlaylistView</class>
   <extends>QTreeView</extends>
   <header>playlistview.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
